middleware/hosts/setup.go

```diff
@@ -80,9 +69,28 @@ func hostsParse(c *caddy.Controller) (Hosts, error) {
 				}
 				return h, c.ArgErr()
 			default:
+				if !h.Fallthrough {
+					line := strings.Join(append([]string{c.Val()}, c.RemainingArgs()...), " ")
+					h.inline = append(h.inline, line)
```

Comment:
Why do you gather raw line instead of parsing the line here?

I also don't think we should have a line with a keyword in front of it (although I made that mistake in the federation middleware).

I.e. `host <hosty-stuff>` for each one you want to add.

Reply:
@miekg Done. The line is parsed in place instead of gathered and parsed later.